.eslintrc.cjs

```diff
@@ -1,23 +1,53 @@
 module.exports = {
+  extends: [
```

Comment:
I wonder, since this is a breaking change, should we try to move to use the new eslint configuration format? https://eslint.org/docs/latest/use/configure/configuration-files-new